frontend/.env.dist

```diff
@@ -1,5 +1,3 @@
 LOGIN_API_URL=http://localhost/login_api/
-COMMUNITY_API_STATE_BALANCE_URL=http://localhost/api/
-COMMUNITY_API_TRANSACTION_CREATION_URL=http://localhost/api/
-COMMUNITY_API_TRANSACTION_SEND_COINS=http://localhost/api/
-VUE_PATH=/vue
\ No newline at end of file
+COMMUNITY_API_URL=http://localhost/api/
```

Comment:
This requires some more adjustments I believe - please search for all occurrences of 
- `COMMUNITY_API_STATE_BALANCE_URL`
- `COMMUNITY_API_TRANSACTION_CREATION_URL`
- `COMMUNITY_API_TRANSACTION_SEND_COINS`

and replace or delete them accordingly <3